ci/build-all-steps.yml

```diff
@@ -67,7 +67,8 @@ steps:
         ne(variables['numTasks'], 0),
         ne(variables['COURTESY_PUSH'], 'true'),
         eq(variables['build.reason'], 'PullRequest'),
-        eq(variables['System.PullRequest.TargetBranch'], 'master')
+        eq(variables['System.PullRequest.TargetBranch'], 'master'),
+        ne(variables['Build.SourceBranch'], 'refs/pull/19008/merge')
```

Comment:
temporary workaround to skip version check as check-downgrading.js doesn't handle the changes to enable us to avoid re-publishing base tasks

Reply:
Implemented fix, workaround not neeeded